dashboard/modules/reporter/reporter_agent.py

```diff
@@ -348,19 +361,25 @@ def _get_load_avg(self):
         per_cpu_load = tuple((round(x / self._cpu_counts[0], 2) for x in load))
         return load, per_cpu_load
 
+    @staticmethod
+    def _compute_speed_from_hist(hist):
+        while len(hist) > 7:
```

Comment:
why is 7 here?

Reply:
Not sure, I just copied what was here already...